src/components/Wishlist/Wishlist.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function Wishlist() {
+  return (
+    <div>
+      <h1> This is Wishlist Page </h1>
+    </div>
+  )
+}
+
+export default Wishlist
```

Comment:
suggestion: don't default export, just export within {} comp Then it will help when you create barrel expo file later.